cpp/src/phonenumbers/geocoding/phonenumber_offline_geocoder.h

```diff
@@ -133,11 +133,11 @@ class PhoneNumberOfflineGeocoder {
 
   // Returns an area-level text description in the given language for the given
   // phone number, or an empty string.
-  // lang is a two-letter lowercase ISO language codes as defined by ISO 639-1.
+  // lang is a two or three-letter lowercase ISO language code as defined by ISO
+  // 639.
   // script is a four-letter titlecase (the first letter is uppercase and the
-  // rest of the letters are lowercase) ISO script codes as defined in ISO
-  // 15924.
-  // region should be a two-letter uppercase ISO country codes as defined by ISO
+  // rest of the letters are lowercase) ISO script code as defined in ISO 15924.
+  // region should be a two-letter uppercase ISO country code as defined by ISO
```

Comment:
This is because you can't pass in a proper language tag, so it's actually the breaking up of a BCP 47 language tag, which does use the ISO codes for region, not CLDR. In practice we use it only for zh-TW vs zh-CN.